site/src/style/variables.less

```diff
@@ -18,3 +18,12 @@
 @jumbotron-color:       white;
 
 @section-vertical-padding: 30px;
+
```

Comment:
> Do we need this?

Subjective!

I just find that the standard bootstrap heading fonts lack enough distinction between H1 and H2:

![screen shot 2015-08-05 at 21 34 59](https://cloud.githubusercontent.com/assets/1098110/9097198/f3398f44-3bb9-11e5-90f7-28c10f7a2fd9.png)

I guess this is why people often vary other aspects of the font such as its weight or caps in order to distinguish between them.